### DISCUSSION TITLE
What is the font name used here?

### DISCUSSION THREAD

**Discussion by superdupermart** [joaotavora/sly#548] [Q&A] [answered]
https://raw.githubusercontent.com/joaotavora/sly/master/doc/animations/backreferences.gif

Not an important question, but what are the font names used here?

**Comment by joaotavora** [comment, marked as answer]
I think it's either "Tamsyn" or "misc-fixed" on Linux.  But I think you can get these fonts for other OS's too.

**Reply by superdupermart** [reply]
I looked it up and it was "Tamsyn". Thanks to you, I know the font name.
